contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -79,7 +85,11 @@ script: |
             echo "REAL=\`which -a ${i}-${prog}-8 | grep -v ${WRAP_DIR}/${i}-${prog} | head -1\`" >> ${WRAP_DIR}/${i}-${prog}
             echo 'export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/faketime/libfaketime.so.1' >> ${WRAP_DIR}/${i}-${prog}
             echo "export FAKETIME=\"$1\"" >> ${WRAP_DIR}/${i}-${prog}
-            echo "exec \"\$REAL\" \"\$@\"" >> $WRAP_DIR/${i}-${prog}
+            if [ "${i:0:9}" = "powerpc64" ]; then
+                echo "exec \"\$REAL\" -mcpu=970 -mtune=power9 \"\$@\"" >> $WRAP_DIR/${i}-${prog}
```

Comment:
I don't know how reliably those get passed everywhere. Also, you mean `CXXFLAGS`...

Reply:
:sweat_smile: yeah I meant CXXFLAGS...

with `depends` you can just modify `hosts/linux.mk`... I'm also not sure about tuning for a specific cpu here (perhaps I'm understanding incorrectly), especially because this will be activated for both big and small endian `HOST`s? Do you have sources for why these flags are needed?

Reply:
We don't want to force these CFLAGS on everyone... just use them for gitian builds so they perform well.

Reply:
Sure, then we can just supply `powerpc64_linux_CFLAGS='-mcpu=970 -mtune=power9' powerpc64le_linux_CFLAGS='-mcpu=970 -mtune=power9'`, but let's do that in a followup :-)

Wondering about the choice of `-mcpu` here... Why PowerPC 970 instead of a more generic `powerpc64` or `powerpc64le`? My reference: https://gcc.gnu.org/onlinedocs/gcc/RS_002f6000-and-PowerPC-Options.html#RS_002f6000-and-PowerPC-Options

Reply:
> Wondering about the choice of -mcpu here... Why PowerPC 970 instead of a more generic powerpc64 or powerpc64le? 

This still atleast needs answering.

Reply:
I don't know. GCC doesn't let me compare `-mcpu` values AFAICT. Presumably generic options wouldn't be optimal for modern CPUs? (even 970 is pretty ancient)

Reply:
I'm told `-mcpu=970` is just powerpc64 + altivec.